app/src/main/java/com/activties/DetailsActivity.kt

```diff
@@ -10,6 +11,11 @@ import com.data.CastList
 import com.data.CreditsResponse
 import com.data.RetrofitClient
 import com.example.myapplication.databinding.ActivityDetailsBinding
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers.IO
+import kotlinx.coroutines.Dispatchers.Main
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
 import retrofit2.Call
 import retrofit2.Callback
 import retrofit2.Response
```

Comment:
Those imports are no longer needed, please always consider optimizing imports after changes, you can also auto imports optimization in the Android Studio.